CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to DuckDuckGo
+
+Thank you for taking the time to contribute to DuckDuckGo! :sparkles
+
+We are pleased to open up the project to you - our community. How can you contribute?
+
+## Share feedback and bug reports
+See [README.md] (https://github.com/duckduckgo/Android/blob/develop/README.md)
```

Comment:
If we use a relative link here, it might work better across branches.

Also, the syntax isn't right here - it doesn't render the link neatly

Reply:
I'm not keen on using relative links for these, I want to point to the most up-to-date instructions. I'll remove the space, I think that will fix the rendering issue.